src/Rector/Class_/AddMethodCallToConstructorForClassesUsingTraitRector.php

```diff
@@ -147,6 +150,7 @@ private function getOrCreateConstructorMethod(Class_ $node): ClassMethod
 
         if (null === $constructor) {
             $constructor = $this->constructorClassMethodFactory->createConstructorClassMethod([], []);
+            $constructor->stmts = [new Expression($this->nodeFactory->createParentConstructWithParams([]))];
             $this->classInsertManipulator->addAsFirstMethod($node, $constructor);
```

Comment:
What if a constructor already exists but doesn't have a parent call?

Reply:
nothing, because constructor will not be null so entire `if` statement will be omitted

Reply:
Oh, I meant if it is a case we want to handle, too (add parent call when constructor already exists)

Reply:
it will do nothing, I've assumed that if the constructor is already added, the author of the code is aware of the parent constructor.